include/nuttx/wireless/ieee80211/ieee80211.h

```diff
@@ -156,692 +375,1228 @@
-#define IEEE80211_AUTH_TRANSACTION(auth) \
-    ((auth)[2] | ((auth)[3] << 8))
-#define IEEE80211_AUTH_STATUS(auth) \
-    ((auth)[4] | ((auth)[5] << 8))
+#define IEEE80211_AUTH_ALG_OPEN   0x0000
+#define IEEE80211_AUTH_ALG_SHARED 0x0001
+#define IEEE80211_AUTH_ALG_LEAP   0x0080
 
-/* Authentication Algorithm Number field (see 7.3.1.1). */
-
-#define IEEE80211_AUTH_ALG_OPEN            0x0000
-#define IEEE80211_AUTH_ALG_SHARED          0x0001
-#define IEEE80211_AUTH_ALG_LEAP            0x0080
-
-/* WEP */
-
-#define IEEE80211_WEP_KEYLEN               5   /* 40bit */
-#define IEEE80211_WEP_NKID                 4   /* number of key ids */
-#define IEEE80211_CHALLENGE_LEN            128
-
-/* WEP header constants */
-
-#define IEEE80211_WEP_IVLEN                3   /* 24bit */
-#define IEEE80211_WEP_KIDLEN               1   /* 1 octet */
-#define IEEE80211_WEP_CRCLEN               4   /* CRC-32 */
-#define IEEE80211_CRC_LEN                  4
-#define IEEE80211_WEP_TOTLEN \
-    (IEEE80211_WEP_IVLEN + IEEE80211_WEP_KIDLEN + IEEE80211_WEP_CRCLEN)
+#define IEEE80211_WEP_KEYLEN  5 /* 40bit */
+#define IEEE80211_WEP_IVLEN   3 /* 24bit */
+#define IEEE80211_WEP_KIDLEN  1 /* 1 octet */
+#define IEEE80211_WEP_CRCLEN  4 /* CRC-32 */
+#define IEEE80211_WEP_TOTLEN  (IEEE80211_WEP_IVLEN + \
+      IEEE80211_WEP_KIDLEN + \
+      IEEE80211_WEP_CRCLEN)
+#define IEEE80211_WEP_NKID    4 /* number of key ids */
 
 /* 802.11i defines an extended IV for use with non-WEP ciphers.
  * When the EXTIV bit is set in the key id byte an additional
  * 4 bytes immediately follow the IV for TKIP.  For CCMP the
  * EXTIV bit is likewise set but the 8 bytes represent the
  * CCMP header rather than IV+extended-IV.
  */
+#define IEEE80211_WEP_EXTIV     0x20
+#define IEEE80211_WEP_EXTIVLEN  4 /* extended IV length */
+#define IEEE80211_WEP_MICLEN    8 /* trailing MIC */
 
-#define IEEE80211_WEP_EXTIV                0x20
-#define IEEE80211_WEP_EXTIVLEN             4    /* extended IV length */
-#define IEEE80211_WEP_MICLEN               8    /* trailing MIC */
+#define IEEE80211_CRC_LEN       4
 
 /* Maximum acceptable MTU is:
- *    IEEE80211_MAX_LEN - WEP overhead - CRC -
- *        QoS overhead - RSN/WPA overhead
+ * IEEE80211_MAX_LEN - WEP overhead - CRC -
+ *  QoS overhead - RSN/WPA overhead
  * Min is arbitrarily chosen > IEEE80211_MIN_LEN.  The default
- * mtu is Ethernet-compatible.
+ * mtu is Ethernet-compatible; it's set by ether_ifattach.
  */
+#define IEEE80211_MTU_MAX  2290
+#define IEEE80211_MTU_MIN  32
 
-#define IEEE80211_MTU_MAX                  2290
-#define IEEE80211_MTU_MIN                  32
-
-#define IEEE80211_MAX_LEN \
-    (2300 + IEEE80211_CRC_LEN + \
+#define IEEE80211_MAX_LEN  (2300 + IEEE80211_CRC_LEN + \
     (IEEE80211_WEP_IVLEN + IEEE80211_WEP_KIDLEN + IEEE80211_WEP_CRCLEN))
 #define IEEE80211_ACK_LEN \
-    (sizeof(struct ieee80211_frame_ack) + IEEE80211_CRC_LEN)
+ (sizeof(struct ieee80211_frame_ack) + IEEE80211_CRC_LEN)
```

Comment:
```suggestion
   (sizeof(struct ieee80211_frame_ack) + IEEE80211_CRC_LEN)
```
just to be the same as in other places